Week06/pi_baris_zirhli.py

```diff
@@ -0,0 +1,19 @@
+import random
+
+def next_pi() -> None:
+    inner, outer = 0, 0
+    while True:
+        randomX, randomY = random.random(), random.random()
```

Comment:
Change `randomX` and `randomY` with `random_x` and `random_y`. This is not JavaScript :)